network.go

```diff
@@ -30,10 +32,38 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-// Introduces constants related to network routes.
+// NetworkInterworkingModel defines the type of container to
+// Virtcontainer network interworking models
+type NetInterworkingModel int
```

Comment:
@sboeuf I would not. The reason is that it is an interworking model. There are cases for macvtap direct and ipvtap and the future veth-tap where we will not be doing any interconnection. We will be directly tapping in. Hence interworking model, instead of interconnection model.